capstone/scripts/convert_s3.py

```diff
@@ -0,0 +1,403 @@
+import base64
+import boto3
+import hashlib
+import json
+import requests
+import tempfile
+from botocore.exceptions import ClientError
+from collections import namedtuple
+
+from capapi.documents import CaseDocument
+from capapi.serializers import NoLoginCaseDocumentSerializer, CaseDocumentSerializer
+from capdb.models import Reporter, VolumeMetadata, CaseMetadata
+
+s3_client = boto3.client("s3")
+
+
+# Doing this with the API because there are many fewer than cases
```

Comment:
Very minor thing: This function would benefit from a docstring `""""` intro comment, like you did for the other functions in this module :).